pkg/openshift/certgen/templates/master/tmp/ansible/azure-ocp-deploy.yml

```diff
@@ -21,6 +21,30 @@
       import_role:
         name: openshift_web_console
 
+    # FIXME - There's a race condition where the infra label isn't applied to
+    #         the infra node during cluster bring up in
+    #         openshift_bootstrap_autoapprover, this is problematic because the
+    #         openshift_prometheus role checks to ensure there is a node with
```

Comment:
Is there an issue in openshift-ansible to track fixing this?

Reply:
Yes https://github.com/openshift/openshift-ansible/issues/8028